### PR TITLE
Announce 2020 schedule

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Blog post about announcing the schedule for PyCon India 2020

### CODE REVIEW

File: content/blog/2020-announce-final-schedule.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+As we are releasing the talk and keynotes schedule, be sure to look out for the release of workshops and information about many more events in the coming days on [Twitter](https://twitter.com/pyconindia) and on [our mailing list](https://mail.python.org/mailman/listinfo/inpycon). Our reharsal team is working double time to get all the speakers in top shape for making this year's PyCon India experience amazing for each and every one of you.
+
+As a final note, we've hit have 1100+ registration and are nearly out of our regular tickets. Be sure to grab yours before stocks last as tickets to the conference are limited. If you haven't bought your ticket yet, the only thing you be left with is regret when they all would go out of stock! **So be sure to buy your PyCon India ticket now.**
```
Comment (Sc0rpi0n101) [inline content/blog/2020-announce-final-schedule.md]: Add a link to the tickets?

File: content/blog/2020-announce-final-schedule.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+As we are releasing the talk and keynotes schedule, be sure to look out for the release of workshops and information about many more events in the coming days on [Twitter](https://twitter.com/pyconindia) and on [our mailing list](https://mail.python.org/mailman/listinfo/inpycon). Our reharsal team is working double time to get all the speakers in top shape for making this year's PyCon India experience amazing for each and every one of you.
+
+As a final note, we are nearly out of our regular tickets. Be sure to grab yours before stocks last as tickets to the conference are limited. If you haven't bought your ticket yet, the only thing you be left with is regret when they all would go out of stock! **So be sure to buy your PyCon India ticket now.**
```
Comment (Sc0rpi0n101) [inline content/blog/2020-announce-final-schedule.md:14]: Add a ticket link?

**Review by Sc0rpi0n101 — COMMENTED** [review]
Add a link to the scbedule somewhere?

File: content/blog/2020-announce-final-schedule.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+2020 has been a rollercoaster for all of us. Our team had setbacks, we pushed through them and what a ride it has been for all of us helping to make PyCon India 2020 a huge success.
+
+The sheer amount of proposals we received from you all over the last couple of days exceeded all our wildest expectations. The review committee along with all the volunteers worked quite hard to make the deadlines work. The schedule has been released a day before the deadline which signals us that we are on track as we move forward according to the plan. All these efforts go on to accomplish one final outcome. An awesome PyCon India keeping you at its epicentre!
+
+As we are releasing the talk and keynotes schedule, be sure to look out for the release of workshops and information about many more events in the coming days on [Twitter](https://twitter.com/pyconindia) and on [our mailing list](https://mail.python.org/mailman/listinfo/inpycon). Our reharsal team is working double time to get all the speakers in top shape for making this year's PyCon India experience amazing for each and every one of you.
+
+Checkout the PyCon India 2020 schedule - https://in.pycon.org/2020/schedule/
```
Comment (Sc0rpi0n101) [inline content/blog/2020-announce-final-schedule.md:14]: This doesn't work as a link in the build, just plain text

File: content/blog/2020-announce-final-schedule.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Title: We have liftoff! The final schedule is out!
+Date: 2020-09-13 22:20
```
Comment (sayanchowdhury) [inline content/blog/2020-announce-final-schedule.md:2]: 2020-09-14 00:00

File: content/blog/2020-announce-final-schedule.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+Title: We have liftoff! The final schedule is out!
+Date: 2020-09-13 22:20 
+Author: Vipul Gupta
+Category: 2020 
+Slug: announce-2020-schedule
+Summary: PyCon India 2020 releases it schedule this time having 4 tracks to the conference packed to the brim with amazing talks and 6 keynote for its next online iteration this year!
```
Comment (sayanchowdhury) [inline content/blog/2020-announce-final-schedule.md:6]: 5 tracks

Comment (sayanchowdhury) [reply content/blog/2020-announce-final-schedule.md:6]: we should say it 5 stages.

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

**Review by sayanchowdhury — APPROVED** [review]

File: content/blog/2020-announce-final-schedule.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+Title: We have liftoff! The final schedule is out!
+Date: 2020-09-14 00:00
+Author: Vipul Gupta
+Category: 2020 
+Slug: announce-2020-schedule
```
Comment (palnabarun) [inline content/blog/2020-announce-final-schedule.md:5]:
```suggestion
Slug: announce-schedule
```

**Review by palnabarun — APPROVED** [review]

**Pull request merged**